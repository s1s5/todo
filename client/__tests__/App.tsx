import App from '../App'

import React from 'react';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
        .create(<App/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});



