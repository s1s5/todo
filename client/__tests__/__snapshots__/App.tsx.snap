// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    Hello, world.
  </h1>
  <span>
    0
     - 
    1
     - 
    1
  </span>
</div>
`;
